cloudman/helmsman/clients/helm_client.py

```diff
@@ -114,14 +116,17 @@ def history(self, release_name):
         data = helpers.run_list_command(["helm", "history", release_name])
         return data
 
-    def rollback(self, release_name, revision=None):
+    def rollback(self, release_name, revision=None, namespace=None):
```

Comment:
@almahmoud Namespace should be mandatory for helm3?

Reply:
We could make it mandatory everywhere. The only place it might not make sense is list.

Reply:
Sounds good.